Neos.ContentRepository/Classes/Migration/Service/NodeTransformation.php

```diff
@@ -85,9 +85,11 @@ protected function buildTransformationObject($transformationConfiguration)
         $transformationClassName = $this->resolveTransformationClassName($transformationConfiguration['type']);
         $transformation = new $transformationClassName();
 
-        foreach ($transformationConfiguration['settings'] as $settingName => $settingValue) {
-            if (!ObjectAccess::setProperty($transformation, $settingName, $settingValue)) {
-                throw new MigrationException('Cannot set setting "' . $settingName . '" on transformation "' . $transformationClassName . '" , check your configuration.', 1343293094);
+        if (is_array($transformationConfiguration)) {
```

Comment:
That doesn't fix the issue completely, better: `isset($transformationConfiguration['settings]) && is_array($transformationConfiguration['settings])`